cs/planet.go

```diff
@@ -245,11 +244,21 @@ func (p *Planet) randomize(rules *Rules) {
 	p.BaseHab = p.Hab
 	p.TerraformedAmount = Hab{}
 
-	// create minconc from min to max (31 to 121)
+	// These two variables are the shape of the normal distribution
+	// based on comparing it with Stars! output
+	mean := 80.0
+	variance := 20.0
+
+	// These two are the min and max of the minerals to be returned,
+	// They clamp the results
+	mMin := 1
+	mMax := 126
```

Comment:
can we set these to `rules.MinStartingMineralConcentration` and `rules.MaxStartingMineralConcentration`